### PR TITLE
Update fuse csb zstream config and remove FIPS config

### PR DESCRIPTION
**Pull request by smpsgill**

### CODE REVIEW

File: ci-operator/config/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main__csb-ocp4.16-lp-interop.yaml
Comment (smongiar) [inline ci-operator/config/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main__csb-ocp4.16-lp-interop.yaml:1]: Maybe I'm wrong, but the chron expression looks like “_At 23:00 on day-of-month 31 in February._” ... Does it never run?

File: ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml
Comment (smongiar) [inline ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml:1]: What's the difference between AWS and IBM cloud? Is the last one another closed environment to the internal network?
Another question is: do we not test ROSA anymore?

File: ci-operator/config/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main__csb-ocp4.16-lp-interop.yaml
Comment (Simranpal) [reply ci-operator/config/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main__csb-ocp4.16-lp-interop.yaml:1]: @smongiar the cron is set intentionally as we don't want to use the cron. We run the tests via a trigger and having cron value set is mandatory for a valid config

File: ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml
Comment (Simranpal) [reply ci-operator/jobs/jboss-fuse/csb-tnb-tests-container/jboss-fuse-csb-tnb-tests-container-main-periodics.yaml:1]: @smongiar I sent email with some of the info

**Comment by smongiar** [comment]
//pj-rehearse

**Comment by Simranpal** [comment]
/pj-rehearse periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-ibmcloud-latest

**Comment by smongiar** [comment]
@Simranpal  as you can see it's ok with the ibm-cloud cluster [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/59595/rehearse-59595-periodic-ci-jboss-fuse-csb-tnb-tests-container-main-csb-ocp4.17-lp-interop-csb-interop-ibmcloud-latest/1867033060004859904)

**Pull request merged**